ci/requirements/doc.yml

```diff
@@ -8,6 +8,8 @@ dependencies:
   - bottleneck
   - cartopy
   - cfgrib
+  - kerchunk
+  - s3fs
```

Comment:
We don't seem to refer to any s3 paths

Reply:
It was for the `remote_protocol` key. I'll remove this as well.